src/main/java/com/conveyal/datatools/editor/utils/JacksonSerializers.java

```diff
@@ -1,6 +1,5 @@
 package com.conveyal.datatools.editor.utils;
```

Comment:
I am curious if this class could be removed in favor of existing class `com.conveyal.gtfs.util.json.JacksonSerializers`, which seems to contain similar classes?

Reply:
@binh-dam-ibigroup `com.conveyal.datatools.editor.utils.JacksonSerializers.java` uses the date format `YYYYMMDD` (`DateTimeFormatter.BASIC_ISO_DATE`) where as `com.conveyal.gtfs.util.json.JacksonSerializers.java` uses `yyyy-MM-dd`. This difference could cause issues.

Reply:
The `com.conveyal.gtfs.util.json.JacksonSerializers` classes are all within gtfs-lib. And it appears these are only applicable when using the validate command line option, which I don't think datatools uses. That being said the remaining classes of `com.conveyal.datatools.editor.utils.JacksonSerializers` appear to only be used within `com.conveyal.datatools.manager.utils.json.JsonManager`. Since that is the case, I'd recommend at the very least moving the JacksonSerializers class to that package if not just inlining the classes within `com.conveyal.datatools.manager.utils.json.JsonManager`.

Reply:
Moved the JacksonSerializers class to `com.conveyal.datatools.manager.utils.json.JsonManager`